plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -113,9 +122,7 @@ class SentryPlugin : Plugin<Project> {
                         getPackageBundleTask(project, variant.name)
                     }
                 } else {
-                    project.logger.info(
-                        "[sentry] isMinifyEnabled is false for variant ${variant.name}."
-                    )
+                    project.logger.info { "isMinifyEnabled is false for variant ${variant.name}." }
```

Comment:
Why you removed `[sentry]` from the logging message? The tasks still have it.

Reply:
It's our custom [extension function](https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/util/SentryLogging.kt) which automatically prepends all logs with `[sentry]`. I will update the SentryPluginTest in a separate PR then, thanks for pointing

Reply:
Thanks for pointing out, I forgot about it 👍